paseos/tests/init_test.py

```diff
@@ -4,27 +4,28 @@
 
 sys.path.append("../..")
```

Comment:
It tries to instantiate paseos twice. I guess this is on purpose. Please, just confirm this.

Reply:
yea it's a bug because we import twice, I think? Not sure how to fix rn but as it is only in the tests (and doesn't break them) I have ignored it for now

Reply:
Maybe, open an issue to check this in the future.

Reply:
#23 please add reproduction steps if possible 🙏